packages/twenty-server/src/modules/calendar/calendar-event-import-manager/drivers/google-calendar/services/google-calendar-get-events.service.ts

```diff
@@ -90,4 +77,34 @@ export class GoogleCalendarGetEventsService {
       nextSyncCursor: nextSyncToken || '',
     };
   }
+
+  private handleError(error: GaxiosError) {
+    if (
+      error.code &&
+      [
+        'ECONNRESET',
+        'ENOTFOUND',
+        'ECONNABORTED',
+        'ETIMEDOUT',
+        'ERR_NETWORK',
+      ].includes(error.code)
+    ) {
+      throw parseGaxiosError(error);
+    }
+    if (error.response?.status !== 410) {
```

Comment:
this if condition needs some details to be added. Why do we ignore 410? Also, it's preferrable to use a if (edgeCase) { return } pattern instead of if (!edgeCase) which leads to nested if / elses that are harder to read